tests/integration/_common.py

```diff
@@ -131,3 +132,12 @@ def run_action_stdout(self, cmdline: List) -> Tuple[str, str]:
         sys.stderr = __stderr__
 
         return stdout, stderr
+
+
+def get_executable_path(name):
```

Comment:
I had better luck using sys
executable here so the same interpreter and venv running the tests would be used